tools/c7n_mailer/c7n_mailer/utils.py

```diff
@@ -90,6 +90,7 @@ def get_message_subject(sqs_message):
 def setup_defaults(config):
     config.setdefault('region', 'us-east-1')
     config.setdefault('ses_region', config.get('region'))
+    config.setdefault('runtime', 'python2.7' if str is bytes else 'python3.6')
```

Comment:
thats a little too implicit, let's do an explicit default. if two people run this i wouldn't want it flopping runtimes.

Reply:
Per https://github.com/capitalone/cloud-custodian/pull/1417#discussion_r135745163, I think the default needs to be the _only_ option: to upload whatever the local runtime is, so that bundled deps are as expected. Otherwise we end up trying to run py27 code under py36 and vice versa, which is lucky to work in more than trivial cases.